fix: Vercel 빌드 오류 - Prisma Client 미생성 및 datasource url 누락

## 문제 설명

Vercel 배포 시 빌드 오류 발생.

```
Error: Turbopack build failed with 1 errors:
./lib/prisma.ts:2:1
Module not found: Can't resolve '@/lib/generated/prisma/client'
```

## 원인

1. **`prisma generate` 미실행**: `package.json`의 `build` 스크립트에 `prisma generate`가 없어 Vercel 빌드 시 Prisma Client 파일이 생성되지 않음
2. **`schema.prisma` datasource `url` 누락**: `datasource db` 블록에 `url = env("DATABASE_URL")`이 없어 DB 연

## 추가 확인사항

- Vercel Dashboard → Project → Settings → Environment Variables 에 `DATABASE_URL` 등록 필요

--- a/prisma/schema.prisma
+++ b/prisma/schema.prisma
@@ -5,7 +5,6 @@ generator client {
 
 datasource db {
   provider  = "postgresql"
-  url       = env("DATABASE_URL")
 }
 
 // ============================================

